spec/fortnox/api/repositories/base_spec.rb

```diff
@@ -221,19 +221,19 @@ def refresh_token=; end
       before do
         Fortnox::API.configure do |config|
           config.token_stores = {
-              default: Class.new do
-              def access_token
-                ''
-              end
+            default: Class.new do
+                       def access_token
+                         ''
+                       end
 
-              def refresh_token
-                ''
-              end
+                       def refresh_token
+                         ''
+                       end
 
-              def access_token=; end
+                       def access_token=; end
 
-              def refresh_token=; end
-            end.new
+                       def refresh_token=; end
+                     end.new
```

Comment:
Why not just make it a module and skip the need for `.new`?

Reply:
Then we would need to stub it to not leak the module between tests. Better to just have it as an anonymous class as it is...